index.js

```diff
@@ -50,6 +49,27 @@ module.exports = {
       fs.unlink(path.join(testAppPath, '.travis.yml')),
     ];
 
+    /**
+     * Setup root package.json scripts based on the packager
+     */
+    tasks.push(
+      (async () => {
+        let packageJson = path.join(options.target, 'package.json');
+        let json = await fs.readJSON(packageJson);
+
+        json.scripts = scripts(options);
+
+        await fs.writeFile(packageJson, JSON.stringify(json, null, 2));
+      })()
+    );
+
+    if (options.pnpm) {
+      let content =
+        `packages:\n` + `  - '${addonInfo.location}'\n` + `  - '${testAppInfo.location}'\n`;
+
+      await fs.writeFile(path.join(options.target, 'pnpm-workspace.yaml'), content);
```

Comment:
Can we extract this into its own method, with a meaningful name, so it is easier to understand what this doing. Also, you could just push it into `tasks`, so all async tasks can operate in parallel.

Reply:
done!